src/resources/employees.js

```diff
@@ -0,0 +1,66 @@
+const express = require('express');
```

Comment:
this should be imported with `import...from`, EVERY library should be imported with that sentence, only .json files should be imported with required

```suggestion
import express from 'express';
```